### PR TITLE
908 - IdsLookup autocomplete popup position

### LINKED ISSUE #908: IdsLookup: Auto complete popup in wrong place
**Describe the bug**
The autocomplete popup; pops up in the wrong place.

**To Reproduce**

1. Go to https://main.wc.design.infor.com/ids-lookup/example.html
2. Resize page s onot much room on the bottom
3. Click on  Autocomplete Lookup
4. Type 10 and wait so it opens
5. Type 3 See error -> in wrong spot

**Expected behavior**
Should open below if it fits or above and next to the input

**Screen shots**
Screen Shot 2022-09-01 at 1 34 52 PM

### PR DESCRIPTION
**Pull request by andriivarhanov**
**Explain the details for making this change. What existing problem does the pull request solve?**
This PR fixes the autocomplete popup position when typing and the results are changing by adding popup's `place` method in `keydownend` event

**Related github/jira issue (required)**:
Closes #908 

**Steps necessary to review your pull request (required)**:
- pull and build
- go to http://localhost:4300/ids-lookup/example.html
- Resize the page so there is not much room on the bottom of Autocomplete Lookup example
- Click on Autocomplete Lookup
- Type 10 and wait so it opens
- Type 3, see the popup is in correct position
- go to http://localhost:4300/ids-input/autocomplete.html
- add some top margin to the `ids-layout-grid` (200px)
- Resize the page
- Type 'nd'
- Type 'ndi', see the popup is in correct position

**Included in this Pull Request**:
~- [ ] An e2e or functional test for the bug or feature.~
- [x] A note to the change log.

### CODE REVIEW

File: doc/CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,8 @@
 - `[Charts]` Added the ability to rotate x axis labels in axis charts like line and bar. ([#738](https://github.com/infor-design/enterprise-wc/pull/738))
 - `[Form]` Added new 'IdsForm' component. ([#357](https://github.com/infor-design/enterprise-wc/pull/357))
 - `[Lookup]` Fixed the dirty tacker was not able to reset. ([#871](https://github.com/infor-design/enterprise-wc/issues/871))
-- `[Pager]` Fixed the direction of the icons in RTL mode ([#865](https://github.com/infor-design/enterprise-wc/issues/865))
+- `[Pager]` Fixed the direction of the icons in RTL mode. ([#865](https://github.com/infor-design/enterprise-wc/issues/865))
+- `[Autocomplete]` Fixed the popup position when autocomplete results changing. ([#908](https://github.com/infor-design/enterprise-wc/issues/908))
```
Comment (tmcconechy) [inline doc/CHANGELOG.md:14]: Can we keep this in alpha order pls?

**Review by tmcconechy — COMMENTED** [review]
Fix looks good. Minor thing with the order. Also i see the layout of the X icon could be improved. Maybe we dont need this rule here for lookup:

Screen Shot 2022-09-09 at 10 25 37 AM

**Comment by andriivarhanov** [comment]
@tmcconechy thanks for the feedback, I've added alphabetical order to the changelog and removed margin for the clearable button

**Review by EdwardCoyle — APPROVED** [review]
Looks good, and makes the Popup more efficient.  Thanks @andriivarhanov!

**Pull request merged**